readme.md

```diff
@@ -116,6 +116,10 @@ Change the `?username=` value to your GitHub username.
 > **Note**
 > Available ranks are S+ (top 1%), S (top 25%), A++ (top 45%), A+ (top 60%), and B+ (everyone). The values are calculated by using the [cumulative distribution function](https://en.wikipedia.org/wiki/Cumulative_distribution_function) using commits, contributions, issues, stars, pull requests, followers, and owned repositories. The implementation can be investigated at [src/calculateRank.js](./src/calculateRank.js).
 
+### Include statistics from private repositories
+
+We do not have permission to see your private repositories. Therefore, we cannot include statistics like stars, commits and pull requests for these private repositories on the public Vercel instance. If you also want to show these private statistics on the stats card, you must [deploy your own Vercel instance](#deploy-on-your-own).
+
```

Comment:
Looks like this change duplicates warning that i have added in #2776. I think that only one note about that should be left. You should merge them or remove any.

Reply:
Your right, I updated this pull request. The language private deploy note now also is a warning for consistency.